.github/workflows/check-tabs.yml

```diff
@@ -0,0 +1,25 @@
+name: Check for Tabs in animint.js
+
+on:
+  push:
+    branches: [main, master]
+  pull_request:
+    branches: [main, master]
+
+jobs:
+  check-tabs:
+    runs-on: ubuntu-latest
+    name: Check animint.js for tabs
```

Comment:
it is a good idea to have linting be a separate CI job like this.
but it is redundant with the testthat file below.
please delete one or the other.
If you keep this file, can lintr package be used?
see data.table for an example: https://github.com/Rdatatable/data.table/blob/master/.github/workflows/code-quality.yaml